element.go

```diff
@@ -158,6 +164,16 @@ func (el *Element) SelectE(selectors []string) error {
 	return err
 }
 
+// AttributeE is similar to the method Attribute
+func (el *Element) AttributeE(name string) (string, error) {
+	attr, err := el.EvalE(true, "(n) => this.getAttribute(n)", Array{name})
+	if err != nil {
+		return "", err
+	}
+
+	return attr.Value.Str, nil
```

Comment:
The specs say when the attribute doesn't exist, it should return `null`:

https://developer.mozilla.org/en-US/docs/Web/API/Element/getAttribute#Non-existing_attributes

We need to have a way to let user know if the attribute exists. Some attributes can be an empty string but they do exist.

I suggest to return `*string`, in golang `*string` can be `nil`. Or return `(string, bool, error)`, which do you think is better?